frontend/components/receipts/utils/order/pegged-order-info.spec.tsx

```diff
@@ -0,0 +1,36 @@
+import { render, screen } from '@testing-library/react'
+import { PeggedOrderInfo } from './pegged-order-info'
+import { PeggedOrder, PeggedReference } from '@vegaprotocol/types'
+
+describe('PeggedOrderInfo', () => {
+  // 1114-RCPT-026 I can see the offset price
+  // 1114-RCPT-027 I can see the reference price
+  const marketId = 'someMarketId'
+
+  it('should render without crashing', () => {
+    const peggedOrder: PeggedOrder = {
+      offset: '12',
+      reference: PeggedReference.PEGGED_REFERENCE_BEST_BID
+    }
+    render(<PeggedOrderInfo peggedOrder={peggedOrder} marketId={marketId} />)
+    expect(screen.getByTestId('price-with-tooltip')).toBeInTheDocument()
+  })
+
+  it('should display the offset price correctly', () => {
+    const peggedOrder: PeggedOrder = {
+      offset: '12',
+      reference: PeggedReference.PEGGED_REFERENCE_BEST_BID
+    }
+    render(<PeggedOrderInfo peggedOrder={peggedOrder} marketId={marketId} />)
+    expect(screen.getByText('12')).toBeInTheDocument()
```

Comment:
In other tests of this type we are request the rows from the table and asserting on those. I think it would be a good idea to follow the same pattern here.